docs/intro.md

```diff
@@ -5,25 +5,37 @@ title: Introduction
 
+In plain simple words, Jitsi allows you to interact with other people though video and audio using your browser, your phone or a desktop app, and brings you all kind of related useful functionalities like text chat, screen sharing, and video recording. All of this wrapped in a friendly interface and without forgetting your security.
+
+
 ## About this handbook
 
-This handbook aims to be the one-stop-shop for all Jitsi documentation.
+This handbook aims to be the one-stop-shop for all Jitsi documentation. 
+
+Whether you are a developer seeking to integrate Jitsi in one of your apps, a more experienced user looking for a specific configuration setup, or even if you are newcomer and have not experience with Jitsi at all, this Handbook will try to help you get there. 
+
+:::note this is a work in progress.
+Notice that some topics or sections may be vague or not as accurate as you may need. We try very hard to keep changes reflected in the docs, but overall the Jitsi Docs Project it's still quite young. 
 
-:::note It's work in progress.
-If you want to help out please create a **Pull Request** in our [Github Repo](https://github.com/jitsi/handbook)!
+- If you want to help out please consider creating a **Pull Request** in our [Github Repo](https://github.com/jitsi/handbook). No contribution is small!
+
+- Not found what you wanted? Post your question in the [Jitsi Comunity Forums](https://community.jitsi.org/).
 :::
 
-The content is divided in 3 main areas:
+That being said... lets go!
+
+## How to read the Handbook
+
+The content is divided in self descriptive sections. Depending of your needs you should start exploring the parts you feel will help you the most. But just in case you don't know where to start, here's a clue of what each topic is about. 
+
+* [The User guide](user-guide/user-guide.md). Designed for helping users to better
+understand all the available features and how to use them. If you are just arriving to the Jitsi party and want to have a feel of what is capable of, this is a good place to start reading.
 
-* [User guide](user-guide/user-guide.md): Designed to help users of the service, to better
-understand all the available features and how to use them.
+* [Developer guide](dev-guide/dev-guide.md): Everything you need to know if you want to integrate the Jitsi Meet API / SDK in one of your products or if you're planing to contribute to some of the Jitsi projects by developing new features or fixing bugs.
 
-* [Developer guide](dev-guide/dev-guide.md): Designed to help developers who want to either
-integrate the Jitsi Meet API / SDK in their products or want to improve Jitsi Meet
-itself by developing new features or fixing bugs.
+* [Self-Hosting guide](devops-guide/devops-guide.md): The place for folks wanting to self-host their own Jitsi service. If you are a system administrator who wants to learn how to deploy, operate and scale Jitsi properly, this is the section you must check out first. 
```

Comment:
I would also shorten it to:
`Self-Hosting Guide: Contains information on how to set up your own Jitsi Server and additional configuration options as well.`